lib/data/repository/recovery_words_repository_impl.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:fluffychat/data/datasource/recovery_words_data_source.dart';
+import 'package:fluffychat/data/model/recovery_words_json.dart';
+import 'package:fluffychat/domain/repository/recovery_words_repository.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/domain/model/recovery_words/recovery_words.dart';
+
+class RecoveryWordsRepositoryImpl implements RecoveryWordsRepository {
+  final RecoveryWordsDataSource recoveryWordsDataSource = getIt.get<RecoveryWordsDataSource>();

```

Comment:
use `final recoveryWordsDataSource = getIt.get<RecoveryWordsDataSource>();` for shorter version